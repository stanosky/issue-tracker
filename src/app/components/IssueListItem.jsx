import { Link } from "react-router-dom";
import React from "react";

export const IssueListItem = ({ id, name }) => (
  <Link to={`/issue/${id}`}>
    <div className="card p-2 mt-2">
      <span>{name}</span>
    </div>
  </Link>
);
