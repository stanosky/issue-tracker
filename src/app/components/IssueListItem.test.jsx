import React from "react";
import { shallow } from "enzyme";
import { IssueListItem } from "./IssueListItem";

describe("IssueListItem", () => {
  it("should render without errors", () => {
      const props = {
          id: 'id',
          name: 'name',
      }
      const wrapper = shallow(<IssueListItem {...props} />);
      expect(wrapper).toMatchSnapshot();
  });
});
