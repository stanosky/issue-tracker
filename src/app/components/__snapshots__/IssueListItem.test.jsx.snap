// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IssueListItem should render without errors 1`] = `
<Link
  replace={false}
  to="/issue/id"
>
  <div
    className="card p-2 mt-2"
  >
    <span>
      name
    </span>
  </div>
</Link>
`;
